packages/library-base/src/commonMain/kotlin/io/realm/kotlin/types/RealmInstant.kt

```diff
@@ -73,6 +74,11 @@ public interface RealmInstant : Comparable<RealmInstant> {
             }
             return RealmInstantImpl(s, ns)
         }
+
+        public fun now(): RealmInstant {
+            val internalNow = internalNow()
+            return RealmInstantImpl(internalNow.epochSeconds, internalNow.nanosecondsOfSecond)
```

Comment:
Since `internalNow()` already returns a `RealmInstant` there is no need to create a new instance. We can just return it directly the user